pkg/controllers/utils/client.go

```diff
@@ -29,47 +29,55 @@ import (
 	networkingv1 "github.com/alibaba/hybridnet/pkg/apis/networking/v1"
 )
 
-func ListNetworks(client client.Reader, opts ...client.ListOption) (*networkingv1.NetworkList, error) {
+func ListNetworks(client client.Client, opts ...client.ListOption) (*networkingv1.NetworkList, error) {
```

Comment:
This should not change, cause `Reader` is just part of `mgr.GetClient()`.

Reply:
fixed.